ui/app/mainui/AppMain.qml

```diff
@@ -1111,6 +1111,7 @@ Item {
     }
 
     Component.onCompleted: {
+        fakeLoadingScreenLoader.active = localAppSettings.fakeLoadingScreenEnabled ?? false
```

Comment:
Hmm, this way we will have 2 splash screens; the one in `main.qml` and this one, presented one after the other. You should imo put this `DidYouKnowSplashScreen` into main.qml and only show one of them, based on `root.advancedStore.isFakeLoadingScreenEnabled`.

Reply:
Actually, we could have only one splash screen (this fake one), and in this very place, we could either hide it (for the devel/source builds), or extend its lifetime and start showing the funny messages (in case of production builds), wdyt?

Reply:
I agree with that 100%.
In addition to the code improvement, we will achieve better visualization as there is currently a strange "gap" / "replacement" the user can appreciate when the new component is loaded. The loading image is like frozen for a second and then the next one starts not in the same position where the other finished.

Reply:
Good point! I've updated to use a single splash screen. Please have a look.